### ISSUE TITLE
[469b] Add posibility filter servers by string and filter out empty, full or locked servers

### ISSUE THREAD

**Issue by SeriousBuggie** [OldUnreal/UnrealTournamentPatches#438]
Possible implementation:
![scr_1622891193](https://user-images.githubusercontent.com/70026933/120907246-a6119280-c668-11eb-8583-3ad0a3d4be97.png)

I can suggest some code after #98 will be accepted.

**Comment by SeriousBuggie** [comment]
Duplicate of #489
